### ISSUE TITLE
share backup location with multiple velero instances for cross cluster migration of workload

### ISSUE THREAD

**Issue by debianmaster** [heptio/velero#1899]
**Describe the problem/challenge you have**
*TL;DR . how to dynamically pass --backup-location to a velero restore.*

I have two clusters with velero installed and pointing to their own minio instances.
My aim is to backup on cluster and restore on another cluster. [ But not expose all the backups, expose only selected backup ]

but i would like achieve this [shown in picture] dynamically on demand.

![BiqOperator](https://user-images.githubusercontent.com/1439169/65401331-019ade80-ddfa-11e9-8fea-ca8e44ae5341.png)

i.e.    point velero in cluster 2 to minio in cluster 1 dynamically [without copying data]

**Describe the solution you'd like**
im thinking of minio -to - mino copy of backup . [may be simple to implement, but data is replicated] 
or 
share the credentials of mino to both veleros [ ALL backups from one location is exposed to another, where i want to expose only the choosen backup] 

**Anything else you would like to add:**
What is the best way to achieve this using the existing velero features?

**Environment:**

- Velero version latest
- Kubernetes version 1.15
- Kubernetes installer & version:
- Cloud provider or hardware configuration:  restic

**Comment by skriss** [comment]
@debianmaster one thing you could consider is having 2 `BackupStorageLocations` in cluster 1 (could be 2 different buckets, or 2 different prefixes within the same bucket), and only giving cluster 2 access to one of them.

**Comment by debianmaster** [comment]
thanks @skriss   i think i have got enough clarity on this, closing this.

**Comment by dvaldivia** [comment]
@debianmaster Additionally you can setup replication between the MinIO's with `mc mirror --watch` containers to have continuous replication between the sites if you want the backups to be available in both sites
